egui_demo_lib/src/apps/demo/plot_demo.rs

```diff
@@ -85,7 +65,7 @@ impl PlotDemo {
                 });
             });
 
-            ui.vertical(|ui| {
+            ui.horizontal(|ui| {
```

Comment:
Unfortunately this makes the plot window very wide, which makes it now very nice on phones. Maybe `ui.horizontal_wrapped` ?